src/utils/MetaProSIP.cpp

```diff
@@ -225,7 +225,7 @@ class MetaProSIPInterpolation
       double dxdy = spline.derivatives(xi, 1);
       double yi = spline.eval(xi);
 
-      if (debug >= 100)
+      if (debug)
```

Comment:
ups I agree :) maybe you could remove the code if you touch the files again (or leave your change as is...)